README.md

```diff
@@ -17,6 +35,84 @@ annotations.scaling.exporter.time.end | "hh:mm:ss"
 annotations.scaling.exporter.replica.min  | "integer"
 
 
+```
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    annotations.scaling.exporter.replica.min: "{{ .Values.hpa.annotations.replica_min"
+    annotations.scaling.exporter.time.end: "{{ .Values.hpa.annotations.time_end }}"
+    annotations.scaling.exporter.time.start: "{{ .Values.hpa.annotations.time_start }}"
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: "{{ .Release.Name }}"
+  minReplicas: {{ .Values.hpa.minReplicas }}
+  maxReplicas: {{ .Values.hpa.maxReplicas }}
+  metrics:
+  - type: External
+    external:
+      metricName: "prescale_metric"
+      metricSelector:
+          matchLabels:
+            deployment: "{{ .Release.Name }}"
+      targetValue: 10
+```
+
+*it's important to set the target to 10, the value provided for scale is 11, the addition of pod will be gradual. Currently, the increment is carried out 10% by 10%.*
+
+## Configure prometheus adapter
+
+A configuration example using the prometheus adapter to supply the metric to the Kubernetes cluster. 
+
+```
+  - "metricsQuery": "avg(<<.Series>>{<<.LabelMatchers>>})"
+    "name":
+      "as": "prescale_metric"
+    "resources":
+      "overrides":
+        "namespace":
+          "resource": "namespace"
+    "seriesQuery": "prescale_metric"
+```
+
+## Configure parameters 
+
+You can modify the application's settings to use different annotations or a different port. Configurable variable environments in the chart values are used to do this.
+
+Parameters            | Default values                            | Comment
+---                   | ---                                       | --- 
+Namespace             | "prescaling-exporter"                     | Namespace for PrescalingEvent
+Port                  | "9101"                                    | Application port
+AnnotationEndTime     | "annotations.scaling.exporter.time.end"   | Annotation end in HPA for create metrique
+AnnotationStartTime   | "annotations.scaling.exporter.time.start" | Annotation start in HPA for create metrique
+AnnotationMinReplicas | "annotations.scaling.exporter.replica.min"| Annotation in min HPA for create metrique
```

Comment:
In English, if you use "for + verb", you must always add "ing" to the verb. Ex: for creating metric.

Anyway, I suggest:
```suggestion
AnnotationEndTime     | "annotations.scaling.exporter.time.end"   | Prescaling end time
AnnotationStartTime   | "annotations.scaling.exporter.time.start" | Prescaling start time
AnnotationMinReplicas | "annotations.scaling.exporter.replica.min"| Minimum of desired replicas during the prescaling
```